Formula/cmocka.rb

```diff
@@ -15,12 +15,13 @@ class Cmocka < Formula
 
   def install
     args = std_cmake_args
+    args << "-DWITH_STATIC_LIB=ON" << "-DWITH_CMOCKERY_SUPPORT=ON" << "-DUNIT_TESTING=ON"
```

Comment:
`STATIC_LIB` should always be on (to compile it, it's pretty small)
`CMOCKERY_SUPPORT` is backwards compatibility with the old cmockery test suite
`UNIT_TESTING` should work, but currently fails a test in the `make test` if built in ruby. should be run, currently working on a work around